src/Raven.Server/Documents/ETL/Providers/Queue/Kafka/KafkaEtl.cs

```diff
@@ -21,7 +25,7 @@ public KafkaEtl(Transformation transformation, QueueEtlConfiguration configurati
     {
     }
 
-    private string TransactionalId => $"{Database.DbId}-{Name}";
+    private string TransactionalId => $"{Database.ServerStore.GetClusterTopology().TopologyId}-{Name}".Replace("/", "_");
```

Comment:
Let's calculate it just once in the ctor. This way it will be easier to debug / investigate in case of any issues with that because we'll have it available in the memory dump.

Reply:
done